aws/s3/main.go

```diff
@@ -0,0 +1,139 @@
+package main
+
+import (
+	"context"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+	"github.com/obot-platform/tools/aws/s3/pkg/commands"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+)
+
+func main() {
+	if len(os.Args) < 2 {
+		log.Fatal("usage: s3 <command>")
+	}
+
+	command := os.Args[1]
+	ctx := context.Background()
+	cfg, err := config.LoadDefaultConfig(ctx)
+	if err != nil {
+		log.Fatalf("Failed to load AWS config: %v", err)
+	}
+
+	region := os.Getenv("AWS_REGION")
+	if region == "" {
+		region = "us-east-1"
+	}
+	cfg.Region = region
+
+	client := s3.NewFromConfig(cfg)
+	bucket := os.Getenv("BUCKET_NAME")
+
+	switch command {
+	case "listBuckets":
+		if commands.ListBuckets(ctx, client) != nil {
+			log.Fatal(err)
+		}
```

Comment:
Otherwise, the `err` here is `nil`. It looks like all of the commands in this file have this issue. Sorry for not catching it earlier.
```suggestion
		if err := commands.ListBuckets(ctx, client); err != nil {
			log.Fatal(err)
		}
```